tests/frameworks/OSXShims/include/IwMalloc.h

```diff
@@ -0,0 +1,67 @@
+//******************************************************************************
+//
+// Copyright (c) 2015 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#ifndef __IWMALLOC_H
+#define __IWMALLOC_H
+
+#include <string.h>
+
+#ifdef _WIN32
```

Comment:
I changed it where I could, but TARGET_OS_XXX and WIN32 don't really reach here.

---
In reply to: [70349688](https://github.com/Microsoft/WinObjC/pull/634#discussion_r70349688) [](ancestors = 70349688)